resources/user-agents/apps/amaya.json

```diff
@@ -11,48 +11,37 @@
         "Parent": "DefaultProperties",
         "Comment": "Amaya",
         "Browser": "Amaya",
-        "Browser_Type": "Application",
-        "Frames": "true",
-        "Tables": "true"
+        "Browser_Type": "Application"
       },
       "children": [
         {
           "match": "amaya/7.*",
           "properties": {
-            "Version": "7.0",
-            "MajorVer": "7"
+            "Version": "7.0"
           }
         },
         {
           "match": "amaya/8.*",
           "properties": {
-            "Version": "8.0",
-            "MajorVer": "8",
-            "CssVersion": "2"
```

Comment:
There is no engine defined on this useragent, won't removing the CssVersion property here change how this one parses?  I assume there aren't any tests covering this pattern.

Same for the ones below.